chrome/locale/en-US/zotero/preferences.dtd

```diff
@@ -101,12 +101,14 @@
 
 <!ENTITY zotero.preferences.quickCopy.caption				"Quick Copy">
 <!ENTITY zotero.preferences.quickCopy.defaultOutputFormat	"Default Output Format:">
+<!ENTITY zotero.preferences.quickCopy.locale				"Bibliography Locale:">
```

Comment:
"Language and Text" or "Language and Region" are fine for those contexts, but they're too unwieldy and broad for our purposes. So our options are using the technically correct "Locale" or just calling this "Language". I'm guess I'm OK with the latter — inexact, but it's not like people are going to be confused given the options.